app/Notifications/NewUpload.php

```diff
@@ -42,6 +43,33 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     *
+     * @return bool
+     */
+    public function shouldSend(User $notifiable): bool
+    {
+        $targetGroup = 'json_following_groups';
+
+        // Do not notify the uploader theirself
+        if ($this->torrent->user_id === $notifiable->id) {
+            return false;
+        }
```

Comment:
Can't follow yourself anyways, so we can probably remove this, similarly to how you've done elsewhere.

Reply:
Removed self part.